locales/en/filters.json

```diff
@@ -213,5 +213,21 @@
 	},
 	{
 		"hostlist.60.description": "Blocks Xiaomi native broadband trackers that track your activity."
```

Comment:
why there is no `hostlist.60.name`? 

why does this description differ from the metadata's one?
"Blocks Xiaomi native broadband tracker***s*** that track your activity"

Reply:
it means that those strings were not uploaded to crowdin (since they were removed during the downloading process)